v0/destinations/am/transform.js

```diff
@@ -213,6 +213,21 @@ function responseBuilderSimple(
 
   // 2. get campaign info (only present for JS sdk and http calls)
   const campaign = get(message, "context.campaign") || {};
+  const oldKeys = Object.keys(campaign);
+  // appends utm_ prefix to all the keys of campaign object. For example the `name` key in campaign object will be changed to `utm_name`
+  oldKeys.forEach(oldKey => {
+    delete Object.assign(campaign, { [`utm_${oldKey}`]: campaign[oldKey] })[
```

Comment:
I think its more readable if we break the copy and deletion of keys into 2 separate operations.
Also I think, `campaign.name` should be mapped to `utm_campaign` as while creating the campaign object in JS SDK, we parse the query string like this only - `utm_campaign` to `campaign.name`. Rest are fine.